tests/constraints-base.in

```
@@ -0,0 +1,3 @@
+# Known limitations for indirect/transitive dependencies.
+
+rstcheck < 6 # # rstcheck 6.x has problem with rstcheck.core triggered by include files w/ sphinx directives https://github.com/rstcheck/rstcheck-core/issues/3
```

Accidental double hash?
```suggestion
rstcheck < 6  # rstcheck 6.x has problem with rstcheck.core triggered by include files w/ sphinx directives https://github.com/rstcheck/rstcheck-core/issues/3
```